### ISSUE TITLE
Changes count is incorrect

### ISSUE THREAD

**Issue by macias** [openstreetmap/iD#9284]
### How to reproduce the issue?

1. Split a line -- change counter 3, should be 1.
2. Change property of part of the above line -- change counter still 3, should be 2.

Of course after many changes the counter is so off that I am not sure if I didn't do some damage to OSM data by accident (like pulling some entity on the map).

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.21.1

### Which browsers are you seeing this problem on?

Firefox

**Comment by mmd-osm** [comment]
Changes count represents the number of objects, that need to be uploaded to OSM later on. I assume the way in your example was also part of some relation. If you split the way, you end up with:

1. the original way with fewer nodes --> _creates a new version of the way for upload_
2. a new way with the remainder of the original way's nodes --> _new object for upload_
3. a relation with the new way member (as a result of step 2)  --> _new version of the relation to be uploaded_

Multiple changes of a way (such as changing properties multiple times) would still affect the same way only once. Those changes are all merged into a single new version of an object.

**Comment by rivermont** [comment]
@tyrasd can this be closed?
